src/v/security/authorizer.h

```diff
@@ -41,11 +42,18 @@ class authorizer final {
     // allow operation when no ACL match is found
     using allow_empty_matches = ss::bool_class<struct allow_empty_matches_type>;
 
-    authorizer()
-      : authorizer(allow_empty_matches::no) {}
-
-    explicit authorizer(allow_empty_matches allow)
-      : _allow_empty_matches(allow) {}
+    authorizer(
+      std::function<config::binding<std::vector<ss::sstring>>()> superusers_cb)
+      : authorizer(allow_empty_matches::no, superusers_cb) {}
+
+    explicit authorizer(
+      allow_empty_matches allow,
+      std::function<config::binding<std::vector<ss::sstring>>()> superusers_cb)
```

Comment:
nit: explicit with more than one arg is redundant?

Reply:
This was just me failing to update the `explicit`s when going from a 0 & 1 arg constructors to 1 & 2 arg constructors.  Fixed.